IGI.Lab3/CarPark.WebUI/Models/AccidentViewModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using CarPark.BLL.Models;
+
+namespace CarPark.WebUI.Models
+{
+    public class AccidentViewModel
+    {
+        [Required]
+        public int Id { get; set; }
+
+        [Required(ErrorMessage = "Id not specified!")]
+        public int ContractId { get; set; }
+
+        [Required]
+        public Contract Contract { get; set; }
```

Comment:
Do we need Contract object here?
If yes, can we avoid ContractId property usage?

Reply:
fix